addon/components/bourbon-text-field.js

```diff
@@ -29,28 +37,43 @@ export default TextField.extend({
   }),
 
   focusedElementObserver: observer('autofocus', function() {
-    this.set('isFocused', this.get('autofocus'))
+    this.set('isFocused', this.get('autofocus'));
+  }),
+
+  valueObserver: observer('value', function() {
+    if (this.get('value') && !this.get('noLabel')) {
+      this.set('isNotEmpty', true);
+    } else {
+      this.set('isNotEmpty', false);
+    }
```

Comment:
Since this is just setting a value, can it be a computed property instead?
```javascript
isEmpty: Ember.computed('value', function() {
  return !this.get('value') && this.get('noLabel');
}),

isNotEmpty: Ember.computed.not('isEmpty');
```
I _think_ that would work the same.

Reply:
will do.

Reply:
it works.  i swear i tried it before and failed...but with a little encouragement from alan, it is all good.  :)